microprofile/tests/testing/testng/src/test/java/io/helidon/microprofile/tests/testing/testng/TestMethodLevelMetaAnnotation.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright (c) 2025 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.helidon.microprofile.tests.testing.testng;
+
+import io.helidon.microprofile.testing.AddBean;
+import io.helidon.microprofile.testing.AddConfig;
+import io.helidon.microprofile.testing.AddConfigBlock;
+import io.helidon.microprofile.testing.Configuration;
+import io.helidon.microprofile.testing.testng.HelidonTest;
+import jakarta.inject.Inject;
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+import org.testng.annotations.Test;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+@HelidonTest
+public class TestMethodLevelMetaAnnotation {
+
+    @Inject
+    private MyBean bean;
+
+    @Inject
+    @ConfigProperty(name = "some.key1")
+    private String value1;
+
+    @Inject
+    @ConfigProperty(name = "some.key2")
+    private String value2;
+
+    @Inject
+    @ConfigProperty(name = "some.key")
+    private String someKey;
+
+    @Inject
+    @ConfigProperty(name = "another.key")
+    private String anotherKey;
+
+    @Inject
+    @ConfigProperty(name = "second-key")
+    private String anotherValue;
+
+    @Test // org.testng.annotations.Test is not inheritable and can't be used in meta-annotation
```

Comment:
I added the annotation `Test` directly. As I understand, we can write it as for junit, because it's not inheritable. See [javadoc](https://www.javadoc.io/doc/org.testng/testng/latest/org/testng/annotations/Test.html). 

Junit [annotation Test](https://junit.org/junit5/docs/current/api/org.junit.jupiter.api/org/junit/jupiter/api/Test.html) has `Testable`-parent and can work.

Reply:
That checks out.